src/redist/targets/packaging/windows/clisdk/bundle.wxs

```diff
@@ -121,9 +145,6 @@
         Example:
             dotnet-sdk-3.0.100-alpha1-009719-win-x64.exe /install DOTNETHOME_X64="D:\dotnet\x64" DOTNETHOME_X86="D:\dotnet\x86" /log "installation.log" /quiet /norestart
     -->
-    <Variable Name="DOTNETHOME_X86" Type="string" Value="[ProgramFilesFolder]dotnet" bal:Overridable="yes" />
```

Comment:
Can these still be overridden from the commandline, e.g. if you want to install outside of ProgramFiles?

Reply:
I think so.  My assumption was that if a variable's value isn't defined it can be specified, but I will test that.

Reply:
yeah, the bal:Overridable property used to control that, but it's only available on Variable elements as an extension, so I'm not sure what the engine defaults are when it's just created as part of a search result

Reply:
You're right, the default is not to flow.  I will bring them back without default values